### DISCUSSION TITLE
Proposal to add "vulnerable code only called with trusted input" dismiss reason for Dependabot security alerts

### DISCUSSION THREAD

**Discussion by stevenwdv** [community/community#45513] [Code Security]
### Select Topic Area

Product Feedback

### Body

Currently these are dismiss reasons supported for Dependabot security alerts:
![dismiss reasons](https://user-images.githubusercontent.com/27223270/215060980-aada7862-a533-43b7-b1f8-ca6020961dd0.png)

I've had multiple cases where the vulnerable code is used, but it is only called with trusted input, and thus not vulnerable to the exploit (mostly with RCE/injection-type exploits).
For this, I propose adding another dismiss reason, something like: "**Vulnerable code is only called with trusted input**".

**Comment by erinhav** [comment]
👋 PM for Dependabot Alerts here. How do you feel about the ability to add [optional dismissal comments](https://github.blog/changelog/2022-08-22-dependabot-alerts-optional-dismissal-comment-2/) when dismissing an alert?

**Comment by stevenwdv** [comment]
Hi! That's what I'm using now, but I feel like it doesn't really fit under any of the existing catagories which I still have to choose from, so maybe an extra category would be nice. Alternatively, if many such cases exist, you could consider adding an "Other" category.
